addons/mail/tests/common.py

```diff
@@ -1180,13 +1180,10 @@ def _activate_multi_lang(cls, lang_code='es_ES', layout_arch_db=None, test_recor
             }
         })
 
-    def _generate_attachments_data(self, count, res_model=None, res_id=None, attach_values=None):
+    @classmethod
+    def _generate_attachments_data(cls, count, res_model=None, res_id=None, attach_values=None):
```

Comment:
```suggestion
    @staticmethod
    def _generate_attachments_data(count, res_model=None, res_id=None, attach_values=None):
```

Reply:
Ok. Also made parmeters not optional to simplify calls.